### PR TITLE
⬆️ plugin: update intl to 0.19.0

### PR DESCRIPTION
**Pull request by tajjacob**
upgrade intl to 0.19.0

#freepalestine
#stopgenocide
#ceasefirenow

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# CodeStream ignored files
```
Comment (bw-flagship) [inline .idea/.gitignore]: Remove this file please

**Review by bw-flagship — CHANGES_REQUESTED** [review]

File: .idea/.gitignore (hunk 1 above)
Comment (tajjacob) [reply .idea/.gitignore]: done. please check again.

**Review by bw-flagship — APPROVED** [review]

**Comment by JgomesAT** [comment]
any new about that merge, this is not applied in pub.dev

**Comment by bw-flagship** [comment]
It is released now in version 3.0.4

**Comment by JgomesAT** [comment]
Your work is done, but need the same of FLutter:

Because easy_localization >=3.0.0-nullsafety depends on flutter_localizations from sdk which depends on intl 0.18.1, easy_localization >=3.0.0-nullsafety requires intl 0.18.1.

**Pull request merged**